praw/models/reddit/mixins/messageable.py

```diff
@@ -23,7 +23,13 @@ def message(self, subject, message, from_subreddit=None):
 
            reddit.redditor('spez').message('TEST', 'test message from PRAW')
 
-        To send a message the moderators of ``/r/test``, try:
+        To send a message to ``/u/spez`` from the moderators of ``/r/test``, try:
+
+        .. code:: python
+
+           reddit.redditor('spez').message('TEST', 'test message from /r/test', '/r/test')
```

Comment:
The last argument should simply be `test`. Additionally, since there are more than just two arguments it would be nice if the last one uses a keyword argument to make it clear what it is. As such, can you make  the last argument be`from_subreddit='test'`?